packages/stryker/src/reporters/ClearTextScoreTable.ts

```diff
@@ -0,0 +1,167 @@
+  }
+
+  /**
+   * Adds padding (spaces) to the front and end of a value
+   * @param input The string input
+   * @param targetLength The min length that has to be reached
+   */
+  protected pad(input: string, targetLength: number = this.width): string {
+    // End with space
+    if (!input.endsWith(' ')) {
+      return this.pad(input + ' ', targetLength);
+    }
+    // Start with space
+    else if (!input.startsWith(' ')) {
+      return this.pad(' ' + input, targetLength);
+    }
+    else if (input.length < targetLength) {
+      // Pad rest with spaces in front
+      return spaces(targetLength - input.length) + input;
+    } else {
+      return input;
+    }
+  }
+
+  drawLine(length = this.width): string {
+    if (length) {
+      return '-' + this.drawLine(length - 1);
+    } else {
+      return '';
+    }
+  }
+
+  drawTableCell(score: ScoreResult, ancestorCount: number) {
+    return this.color(score)(this.pad(this.valueFactory(score, ancestorCount)));
+  }
+
+  drawHeader() {
+    return this.pad(this.header);
+  }
+
+  protected color(score: ScoreResult) {
+    return (input: string) => input;
+  }
+}
+
+class MutationScoreColumn extends Column {
+  constructor(rows: ScoreResult) {
+    super('% score', row => row.mutationScore.toFixed(2), rows);
+  }
+  protected color(score: ScoreResult) {
+    if (score.mutationScore >= 80) {
+      return chalk.green;
+    } else if (score.mutationScore >= 60) {
+      return chalk.yellow;
+    } else {
+      return chalk.red;
+    }
+  }
+}
+
+class FileColumn extends Column {
+  constructor(rows: ScoreResult) {
+    super('File', (row, ancestorCount) => spaces(ancestorCount) + (ancestorCount === 0 ? FILES_ROOT_NAME : row.name), rows);
+  }
+  protected pad(input: string, targetLength: number = this.width): string {
+    if (!input.endsWith(' ')) {
+      return this.pad(`${input} `, targetLength);
+    } else if (input.length < targetLength) {
+      return input + spaces(targetLength - input.length);
+    } else {
+      return input;
+    }
+  }
+}
+
+/**
+ * Represents a clear text table for mutation score
+ */
+export default class ClearTextScoreTable {
+
+  private columns: Column[];
+
+  constructor(private score: ScoreResult) {
+    this.columns = [
+      new FileColumn(score),
+      new MutationScoreColumn(score),
+      new Column('# killed', row => row.killed.toString(), score),
+      new Column('# timeout', row => row.timedOut.toString(), score),
+      new Column('# survived', row => row.survived.toString(), score),
+      new Column('# no cvg', row => row.noCoverage.toString(), score),
+      new Column('# error', row => row.errors.toString(), score)
+    ];
+  }
+
+  private drawBorder() {
+    return this.drawPerColumn(column => column.drawLine());
+  }
+
+  private drawHeader() {
+    return this.drawPerColumn(c => c.drawHeader());
+  }
+
+  private drawPerColumn(toDraw: (col: Column) => string) {
+    return this.columns.map(toDraw).join('|') + '|';
```

Comment:
I've shamelessly copied the clear text table of [`nyc`](https://github.com/istanbuljs/nyc)